src/graphtypes/simplegraphs/simplegraph.jl

```diff
@@ -139,6 +139,44 @@ function add_edge!(g::SimpleGraph, e::SimpleGraphEdge)
     return inserted
 end
 
+function add_edges!(g::SimpleGraph, es)
+    T = eltype(g)
+    nvg = nv(g)
+    neg = ne(g)
+    for e in es
+        s, d = T.(Tuple(e))
+        if s < 1 || d < 1
+            error("Nonpositive vertex numner")
+        elseif s == d
+            error("Self loop in edges")
+        end
+        for v in (s, d)
+            if v > nvg
+                resize!(g.fadjlist, v)
+                for i in nvg+1:v
+                    g.fadjlist[i] = Vector{T}()
+                end
+                nvg = v
+            end
+        end
+        append!(g.fadjlist[s], d)
+        append!(g.fadjlist[d], s)
+    end
+    nenew = 0
+    for s in 1:nvg
+        sort!(g.fadjlist[s])
+        if VERSION >= v"0.7.0-DEV.601"
```

Comment:
nonstarter. We don't do version-specific code in LG.

Reply:
why? this is the right way to do it

Reply:
We used to allow this back in the 0.3. / 0.4 days, but it became a mess of spaghetti conditionals and really impeded troubleshooting. We support whatever the release version of Julia is – that is, 0.6.x.